demo/starter/slides.md

```diff
@@ -26,19 +26,16 @@ mdc: true
 
 Presentation slides for developers
 
-<div class="pt-12">
-  <span @click="$slidev.nav.next" class="px-2 py-1 rounded cursor-pointer" flex="~ justify-center items-center gap-2" hover="bg-white bg-opacity-10">
-    Press Space for next page <div class="i-carbon:arrow-right inline-block"/>
-  </span>
+<div @click="$slidev.nav.next" mt-12 py-1 hover:bg="white op-10">
+  Press Space for next page <carbon:arrow-right />
 </div>
 
-<div class="abs-br m-6 flex gap-2">
-  <button @click="$slidev.nav.openInEditor()" title="Open in Editor" class="text-xl slidev-icon-btn opacity-50 !border-none !hover:text-white">
-    <div class="i-carbon:edit" />
+<div abs-br m-6 text-xl>
```

Comment:
The reason to fix class and attributify as to demonstrate both mode are valid. People that wasn't using UnoCSS before might get confused if we use valueless mode up front